README.md

```diff
@@ -4,24 +4,50 @@ This project contains [Helm](https://helm.sh/) charts for [Cluster API](https://
 
 ## Installing CAPI Charts
 
-### [azure-aks-aso](./charts/azure-aks-aso)
-
-To install an AKS cluster for CAPZ using the ASO API, use the following commands:
+This is the foundational step for any cloud provider to install CAPI and then use any charts from this repository:
 
 ```shell
-clusterctl init --infrastructure azure
+helm repo add capi-operator https://kubernetes-sigs.github.io/cluster-api-operator
 helm repo add capi https://mboersma.github.io/cluster-api-charts
-helm install <name> capi/azure-aks-aso -f <my_values.yaml>
+helm repo add jetstack https://charts.jetstack.io --force-update
+helm repo update
+helm install \
+  cert-manager jetstack/cert-manager \
+  --namespace cert-manager \
+  --create-namespace \
+  --set installCRDs=true
 ```
 
-### [azure-managed-cluster](./charts/azure-managed-cluster)
+### Azure
 
-To install an AKS cluster for CAPZ as a ManagedCluster, use the following commands:
+Use this command to install CAPI with the Azure provider (CAPZ) and all feature flags for any of the Azure charts below.
 
 ```shell
-clusterctl init --infrastructure azure
-helm repo add capi https://mboersma.github.io/cluster-api-charts
-helm install <name> capi/azure-managed-cluster -f <my_values.yaml>
+helm install capi-operator capi-operator/cluster-api-operator --create-namespace -n capi-operator-system \
+--set infrastructure="azure" \
+--set manager.featureGates.core.MachinePool="true" \
+--set manager.featureGates.azure.MachinePool="true" \
+--set manager.featureGates.azure.ASOAPI="true" \
+--set manager.featureGates.azure.ClusterResourceSet="true" \
+--set manager.featureGates.azure.ClusterTopology="true" \
+--set manager.cert-manager.enabled="false" \
+--set manager.cert-manager.installCRDs="false"
+```
+
+#### [azure-aks-aso](./charts/azure-aks-aso)
+
+To install an AKS cluster for [CAPZ using the new experimental ASO API](https://capz.sigs.k8s.io/topics/aso.html?highlight=azure%20service%20ope#experimental-aso-api), use the following commands:
+
+```
+helm install capaksaso capi/azure-aks-aso -f <my_values.yaml>
```

Comment:
Could we use something other than `capaksaso` here? My brain is having a hard time decoding that one. I was using the name of the chart `azure-aks-aso` as its local name.

Reply:
We hit an error with the chart when the name spec is more than 9 characters 👎 .  How about `capz-aso`?

Reply:
Oh re-he-heally? Sure, I like `capz-aso` much better, thanks!